doc/convert.sh

```diff
@@ -4,9 +4,21 @@ set -e
 
 for f in $(find $1 -name '*.adoc'); do
   echo $f
+  sed -i 's/\[source, *motoko\]/\[source, motoko\]/g' $f
+  sed -i 's/\[source.no-repl, *motoko\]/\[source, motoko no-repl\]/g' $f
+  sed -i 's/\[source.run, *motoko\]/\[source, motoko run\]/g' $f
+  sed -i 's/\[source, *candid\]/\[source,candid\]/g' $f
+  sed -i 's/\[source, *bnf\]/\[source,bnf\]/g' $f
+  sed -i 's/\[source, *bash\]/\[source,bash\]/g' $f
+  sed -i 's/\[source\#\([a-zA-Z]*\), *motoko\]/\[source, motoko filename=\1\]/g' $f
```

Comment:
The field is `name` instead of `filename`

Reply:
Dooh.
Will fix now.

Reply:
@chenyan-dfinity PTAL